apps/web/util/types.ts

```diff
@@ -17,12 +17,12 @@ export type UploadingFile = {
 export type Config = FirebaseOptions & { password?: string };
 
 export enum Provider {
-	firebase,
-	s3,
-	minio,
-	backblaze,
-	deta,
-	git,
+  firebase,
+  s3,
+  minio,
+  backblaze,
+  deta,
+  git,
 	wasabi,
 	digitalocean,
   cloudflare,
```

Comment:
why did we change this?

Reply:
This was after pulling the owners changes, it was reformatted.